org-formation/_tasks.yaml

```diff
@@ -11,9 +11,14 @@ Types:
   DependsOn: [ OrganizationUpdate ]
   Path: ./005-types/_tasks.yaml
 
+Macros:
```

Comment:
do we really need a folder just for macros?  this lambda seems like it's specifically related to `050-costs` so why not just deploy it from there?

Reply:
I'm trying to treat the macros similar to the types since the stack needs to complete before the macro is available. We might not ever create a second macro, but I think it still deserves its own stage so that we can manage the dependency at the top level in this file.

Reply:
i think that would make sense if the lambda was more general purpose however this lambda is specific to cost so I would prefer to keep them in the same folder so that it's in context.  You can setup dependencies between `update-stack` deployments by using the orgf's [DependOn](https://github.com/org-formation/org-formation-cli/blob/master/docs/task-files.md#update-stacks) parameter.

Reply:
done